### PR TITLE
fix: TOOLS-2971 fix in existing failing test cases

### PR DESCRIPTION
**Pull request by a-spiker**
- Fixing the existing test cases to ensure any further changes in the repo are properly tested.
- Removing the suppress check on the go test command

### CODE REVIEW

File: info/as_parser.go [hunk 1]
```diff
@@ -1148,7 +1148,7 @@ func parseAllXDRConfig(rawMap map[string]string, cmd string) lib.Stats {
 
 	if dcNamesRaw == "" {
 		dcNames = []string{}
-		xdrConfigMap[ConfigDCContext] = struct{}{}
```
Comment (dwelch-spike) [inline info/as_parser.go:1151]: Why change this? Does it cause a bug somewhere else?

**Comment by dwelch-spike** [comment]
@a-spiker For our understanding could you lay out why these changes were made? What issues were they causing?

1. "null" changed to ""
2. struct{}{} -> make(lib.Stats)

File: asconfig/generate_test.go [hunk 2]
```diff
@@ -586,8 +586,8 @@ var namespaceTC = GenerateTC{
 				"storage-engine.direct-files":               false,
 				"storage-engine.disable-odsync":             false,
 				"storage-engine.enable-benchmarks-storage":  false,
-				"storage-engine.encryption-key-file":        "null",
-				"storage-engine.encryption-old-key-file":    "null",
```
Comment (jdogmcsteezy) [inline asconfig/generate_test.go:590]: Rather than editing the input config, which you are doing here, I think you should edit the output config. [Here](https://github.com/aerospike/aerospike-management-lib/blob/master/asconfig/generate.go#L574-L576) is section of code that removes "null" values from the output config. The "null" value is returned from the server and the goal of this code is to turn it into a map struct (lib.Stats) that can be easily marshalled to a yaml file.

Basically, what I think you should do is remove `key-file`, `old-key-file`, etc. from the expected array rather than change what the server response looks like

**Review by jdogmcsteezy — CHANGES_REQUESTED** [review]

**Comment by a-spiker** [comment]
> @a-spiker For our understanding could you lay out why these changes were made? What issues were they causing?
> 

> 2. struct{}{} -> make(lib.Stats)

struct{}{} -> make(lib.Stats)
There is an existing  test case which was failing, which expects dis to be of lib.Stats. Ideally all the sub configs should be instance of lib.Stats for consistent parsing..   It was changed here https://github.com/aerospike/aerospike-management-lib/pull/40/files#diff-3e8a272f186a1070104d40c5f775480b3f913489b147098e3662f7eda1cf8f1cR1149 . if the dc values is empty it was still going to have Len > 0 for the split. 


> 1. "null" changed to ""
As per the review, I have changed the expected values rather than test generated values.

**Review by jdogmcsteezy — APPROVED** [review]

File: info/as_parser.go [hunk 3]
```diff
@@ -1148,7 +1148,7 @@ func parseAllXDRConfig(rawMap map[string]string, cmd string) lib.Stats {
...
+		xdrConfigMap[ConfigDCContext] = make(lib.Stats)
```
Comment (tanmayja) [inline info/as_parser.go:1151]: should we make the same change at line 1173?

**Review by dwelch-spike — APPROVED** [review]
LGTM

**Review by tanmayja — APPROVED** [review]
